app/_src/gateway/install/docker/build-custom-images.md

```diff
@@ -17,19 +17,24 @@ chmod +x docker-entrypoint.sh
 ```
 
 1. Download the {{site.base_gateway}} package:
-    * **Debian**: [.deb]({{ site.links.cloudsmith }}/public/gateway-{{ page.major_minor_version }}/deb/debian/pool/bullseye/main/k/ko/kong-enterprise-edition_{{page.versions.ee}}/kong-enterprise-edition_{{page.versions.ee}}_amd64.deb).
-    * **Ubuntu**: [.deb]({{ site.links.cloudsmith }}/public/gateway-{{ page.major_minor_version }}/deb/ubuntu/pool/jammy/main/k/ko/kong-enterprise-edition_{{page.versions.ee}}/kong-enterprise-edition_{{page.versions.ee}}_amd64.deb).
+    * **Debian**: [.deb]({{ site.links.direct }}/gateway-{{ page.major_minor_version }}/deb/debian/pool/bullseye/main/k/ko/kong-enterprise-edition_{{page.versions.ee}}/kong-enterprise-edition_{{page.versions.ee}}_amd64.deb).
+    {% if_version lte:3.0.x %}
+    * **Ubuntu**: [.deb]({{ site.links.direct }}/gateway-{{ page.major_minor_version }}/deb/ubuntu/pool/focal/main/k/ko/kong-enterprise-edition_{{page.versions.ee}}/kong-enterprise-edition_{{page.versions.ee}}_amd64.deb).
+    {% endif_version %}
+    {% if_version gte:3.1.x %}
+    * **Ubuntu**: [.deb]({{ site.links.direct }}/gateway-{{ page.major_minor_version }}/deb/ubuntu/pool/jammy/main/k/ko/kong-enterprise-edition_{{page.versions.ee}}/kong-enterprise-edition_{{page.versions.ee}}_amd64.deb).
+    {% endif_version %}
     {% comment %}
     not all of the older alpine "packages" met Cloudsmith's definition for what an alpine package must be
     so some are uploaded there as "raw" artifacts instead and must be linked to differently
     {% endcomment %}
-    {% if_version eq:3.0.x %}
-    * **Alpine**: [.apk]({{ site.links.cloudsmith }}/public/gateway-{{ page.major_minor_version }}/raw/names/kong-enterprise-edition-x86_64/versions/{{page.versions.ee}}/kong-enterprise-edition-{{page.versions.ee}}.x86_64.apk.tar.gz)
+    {% if_version lte:3.3.x eq:3.0.x %}
```

Comment:
@curiositycasualty I think the `eq:3.0.x` might be overriding the lte 3.3.x. If this should be for all versions equal to and less than 3.3.x, we can just remove `eq:3.0.x`

Reply:
Alright, I think I have it now:
- https://deploy-preview-6863--kongdocs.netlify.app/gateway/3.4.x/install/docker/build-custom-images/ should not list alpine pkgs at all
- https://deploy-preview-6863--kongdocs.netlify.app/gateway/3.3.x/install/docker/build-custom-images/ should list a valid `.apk` package URL
- https://deploy-preview-6863--kongdocs.netlify.app/gateway/3.0.x/install/docker/build-custom-images/ should list a valid `.apk.tar.gz` package URL
- this page doesn't exist for `lte:2.8.x`